### PR TITLE
Update deposit.md

### PR DESCRIPTION
**Pull request by twynne20**
Change link for Contract docs, the previous link is 404'ing

### CODE REVIEW

**Review by alokeutpal — APPROVED** [review]

File: docs/advanced/deposit.md
Comment (popzxc) [inline docs/advanced/deposit.md:124]: I think the correct link would be `https://github.com/matter-labs/era-contracts/blob/main/docs/Overview.md`.

**Review by popzxc — COMMENTED** [review]
Thank you for the contribution and sorry for a long wait on review -- we're still getting used to developing in the open.

Looks like the file was renamed to `02_deposits.md`, could you please resolve the merge conflict and apply the suggestion? After that, I'll be happy to merge.

File: docs/advanced/deposit.md
Comment (twynne20) [reply docs/advanced/deposit.md:124]: Hi @popzxc,

I've addressed the feedback and created a new pull request based on the latest code from the main branch. I've also updated the link as you recommended.

[New pull request #335 ](https://github.com/matter-labs/zksync-era/pull/335)

I've kept this original PR open for reference, but feel free to close it if the new one looks good.

Thank you for your guidance and patience for a small silly fix 💯

**Pull request closed without merging**
